include/xrpl/protocol/TER.h

```diff
@@ -673,7 +673,8 @@ isTerRetry(TER x) noexcept
 inline bool
 isTesSuccess(TER x) noexcept
 {
-    return (x == tesSUCCESS);
+    // Makes use of TERSubset::operator bool()
+    return !(x);
```

Comment:
I was initially "huh" but then looked at the function name. This is entirely appropriate implementation for `isTesSuccess`; I would not like it elsewhere.